ci/cpu/clang12_release.yml

```
@@ -0,0 +1,29 @@
+include:
+  - local: 'ci/common-ci.yml'
+
+cpu clang12 release deps:
+  extends: .build_deps_common
+  variables:
+    EXTRA_APTGET: "clang-12 libomp-12-dev"
+    COMPILER: clang@12.0.1
+    USE_MKL: "ON"
+    SPACK_ENVIRONMENT: ci/docker/release-cpu.yaml
+    BUILD_IMAGE: $CSCS_REGISTRY_PATH/release-cpu-clang12/build
+
+cpu clang12 release build:
+  extends:
+    - .build_common
+    - .build_for_daint-mc
```

Cuda tests should be run on daint-gpu.
```suggestion
    - .build_for_daint-gpu
```
(Applies to other cuda tests)

---

Fixed